common-services.yml

```diff
@@ -53,6 +53,11 @@ services:
         max-buffer-size: 100m
       driver: json-file
 
+  healthz:
+    image: audius/healthz:latest
```

Comment:
I'm open to changing the tag here or leaving it as-is since healthz will be built manually for the time being.

Reply:
i do think a tag here is better because it's very easy to accidentally push to `latest` by accident or on purpose without realizing that some SPs need another update first. i've also seen an issue before where audius-cli didn't realize the version changed (but maybe we can fix that at the audius-cli level).

can you set to the the tag that's on the latest release and manually push that once? so like `audius/healthz:${TAG:-e4155e996efcb896c152a3f0fe651b46171ba6b8}` and then a docker build with the tag set to `e4155e996efcb896c152a3f0fe651b46171ba6b8`

Reply:
Updated to use the tag for release-v0.4.11 (bc28f396b2606af49c65591119454484b900cf96)